src/selfplay/tournament.cc

```diff
@@ -469,4 +636,26 @@ SelfPlayTournament::~SelfPlayTournament() {
   Wait();
 }
 
+void SelfPlayTournament::SaveResults() {
+  if (kTournamentResultsFile.empty()) return;
+  std::ofstream output(kTournamentResultsFile, std::ios_base::app);
+  auto p1name =
+      player_options_[0][0].Get<std::string>(NetworkFactory::kWeightsId);
+  auto p2name =
+      player_options_[1][0].Get<std::string>(NetworkFactory::kWeightsId);
+
+  output << std::endl;
+  output << "[White \"" << p1name << "\"]" << std::endl;
+  output << "[Black \"" << p2name << "\"]" << std::endl;
+  output << "[Results \"" << tournament_info_.results[0][0] << " "
+         << tournament_info_.results[2][0] << " "
+         << tournament_info_.results[1][0] << "\"]" << std::endl;
+  output << std::endl;
```

Comment:
Isn't the second part only valid for policy tournament?

Reply:
I believe this is valid for any tournament where p1name and p2name are valid for both the white vs black and black vs white case. (I didn't feel like it was worth trying to handle the case where that is false...)

It is fake pgn regardless - no per-game moves, no tracking of openings, just raw results in a format which is consumable by my custom version of ordo...